pkg/apps/utils.jsx

```diff
@@ -56,7 +56,7 @@ export const ProgressBar = ({ size, title, data, ariaLabelledBy }) => {
                 {_("Waiting for other programs to finish using the package manager...")}
             </SplitItem>
             <SplitItem>
-                <Spinner isSVG size="md" />
+                <Spinner size="md" />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230426-130726-0a013b1d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)